### PR TITLE
update zh-translation: docs/concepts/storage/volume-snapshot-classes.md

### PR DESCRIPTION
**Pull request by helight**
update zh-translation: docs/concepts/storage/volume-snapshot-classes.md
ref:https://github.com/k8smeetup/website-tasks/issues/3678

### CODE REVIEW

File: content/zh/docs/concepts/storage/volume-snapshot-classes.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ---
+reviewers:
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volume-snapshot-classes.md]: The reviewers can be safely removed in translated version.

File: content/zh/docs/concepts/storage/volume-snapshot-classes.md [hunk 2]
```diff
@@ -46,47 +51,57 @@ be updated once they are created.
...
+### DeletionPolicy
 
-卷快照类具有一个快照生成器，用于确定配置 VolumeSnapshot 的 CSI 卷插件。 必须指定此字段。
+Volume snapshot classes have a deletionPolicy. It enables you to configure what happens to a `VolumeSnapshotContent` when the `VolumeSnapshot` object it is bound to is to be deleted. The deletionPolicy of a volume snapshot can either be `Retain` or `Delete`. This field must be specified.
+
+If the deletionPolicy is `Delete`, then the underlying storage snapshot will be deleted along with the `VolumeSnapshotContent` object. If the deletionPolicy is `Retain`, then both the underlying snapshot and `VolumeSnapshotContent` remain.
+-->
+### 删除策略 {#deletion-policy}
+
+卷快照类有删除策略。它可以配置一个策略，当 `VolumeSnapshot` 对象被删除时，对它所绑定的 `VolumeSnapshotContent` 可以做什么。卷快照的这个策略可以是 `Retain` 或者 `Delete`。这个策略字段必须明确。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volume-snapshot-classes.md]:
```suggestion
卷快照类具有 `deletetionPolicy` 属性。用户可以配置当所绑定的 `VolumeSnapshot` 对象将被删除时，如何处理 `VolumeSnapshotContent` 对象。卷快照的这个策略可以是 `Retain` 或者 `Delete`。这个策略字段必须指定。
```

File: content/zh/docs/concepts/storage/volume-snapshot-classes.md (hunk 1 above)
Comment (tengqm) [inline content/zh/docs/concepts/storage/volume-snapshot-classes.md]: The reviewers can be safely removed in translated version.

File: content/zh/docs/concepts/storage/volume-snapshot-classes.md [hunk 3]
```diff
@@ -46,47 +51,57 @@ be updated once they are created.
...
 the volume snapshot class. Different parameters may be accepted depending on the
-`snapshotter`.
+`driver`.
 -->
+## 参数 {#parameters}
 
-## 参数
-
-卷快照类具有描述属于卷快照类的卷快照参数。 可根据 `snapshotter` 接受不同的参数。
+卷快照类具有描述属于卷快照类的卷快照参数。 可根据 `driver` 接受不同的参数。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volume-snapshot-classes.md]:
```suggestion
卷快照类具有描述属于该卷快照类的卷快照的参数。 可根据 `driver` 接受不同的参数。
```

File: content/zh/docs/concepts/storage/volume-snapshot-classes.md [hunk 4]
```diff
@@ -46,47 +44,57 @@ be updated once they are created.
...
+### DeletionPolicy
 
-卷快照类具有一个快照生成器，用于确定配置 VolumeSnapshot 的 CSI 卷插件。 必须指定此字段。
+Volume snapshot classes have a deletionPolicy. It enables you to configure what happens to a `VolumeSnapshotContent` when the `VolumeSnapshot` object it is bound to is to be deleted. The deletionPolicy of a volume snapshot can either be `Retain` or `Delete`. This field must be specified.
+
+If the deletionPolicy is `Delete`, then the underlying storage snapshot will be deleted along with the `VolumeSnapshotContent` object. If the deletionPolicy is `Retain`, then both the underlying snapshot and `VolumeSnapshotContent` remain.
+-->
+### 删除策略 {#deletion-policy}
+
+卷快照类具有 `deletetionPolicy` 属性。用户可以配置当所绑定的 `VolumeSnapshot` 对象将被删除时，如何处理 `VolumeSnapshotContent` 对象。卷快照的这个策略可以是 `Retain` 或者 `Delete`。这个策略字段必须指定。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volume-snapshot-classes.md]: deletetionPolicy 拼写错误

Comment (helight) [reply content/zh/docs/concepts/storage/volume-snapshot-classes.md]: got，thanks。

**Pull request merged**